ShowdownAI.Middleware/ShowdownAI.Middleware/Models/Pokemon.cs

```diff
@@ -2,20 +2,31 @@
 {
     public class Pokemon
     {
-        public string Ident { get; set; }
-        public string Details { get; set; }
-        public string Condition { get; set; }
+        public string Ability { get; set; }
         public bool Active { get; set; }
-        public Stats Stats { get; set; } 
-        public List<string> Moves { get; set; }
         public string BaseAbility { get; set; }
+        public bool Commanding { get; set; }
+        public string Condition { get; set; }
+        public string Details { get; set; }
+        public string Ident { get; set; }
         public string Item { get; set; }
+        public List<string> Moves { get; set; }
         public string Pokeball { get; set; }
-        public string Ability { get; set; }
-        public bool Commanding { get; set; }
         public bool Reviving { get; set; }
-        public string TeraType { get; set; }
+        public Stats Stats { get; set; } 
+        public Status Status {get ; set;}
         public string Terastallized { get; set; }
+        public string TeraType { get; set; }
+
+    }
 
+    enum Status
```

Comment:
Possible types found here - https://github.com/smogon/pokemon-showdown-client/blob/1019ac1b59aa34decf0985395e249554f84cd8fb/play.pokemonshowdown.com/src/battle-dex-data.ts#L1058.

Not used C# so not sure if it's entirely standard but an Enum seemed like a good idea here.

Reply:
![image](https://github.com/user-attachments/assets/b69816fa-736f-4d6c-bd4d-2c6aff26caac)

Reply:
Enums are fine. However please create a new file called Status that matches the format here:
https://github.com/ljm862/ShowdownAI/blob/main/ShowdownAI.Middleware/ShowdownAI.Middleware/Models/ShowdownAction.cs
This way the enum will be public and accessible to the other parts of the code as opposed to internal to this Model

Reply:
Done - [move Status enum to seperate file](https://github.com/ljm862/ShowdownAI/pull/1/commits/2e18f1050ec2e864db59aefdd798ee026c9a1150)